### DISCUSSION TITLE
No Kernel modules found

### DISCUSSION THREAD

**Discussion by no-good-usernames-left** [telekom-security/tpotce#1052] [General]
I can't build a new T-pot Instance or install from the ISO because the Kernel modules have moved 

What has gone wrong and how do we fix this?

**Comment by no-good-usernames-left** [comment]
![disks](https://user-images.githubusercontent.com/47146326/161147370-353c5fe3-85e2-4da4-bbf2-a8c05b4bdaa6.PNG)

**Reply by t3chn0m4g3** [reply]
Did you download the latest ISO, updated it a few days ago?

**Reply by no-good-usernames-left** [reply]
The latest ISO release is over a year old - Feb 22, 2021 and as a retry, it's the same one that has issues with the kernel modules 

https://github.com/telekom-security/tpotce/releases

**Reply by t3chn0m4g3** [reply]
No, I swapped it. Described and resolved in #1048.

**Reply by no-good-usernames-left** [reply]
Ah, a published announcement/release would be most helpful in a faster time to rebuild, 

New ISO fetched and it works!

**Reply by no-good-usernames-left** [reply]
t3, can you take a look at this thread?
I need this looked at because the updated t-pot has the management ports going _through_ the honeypots 

https://github.com/telekom-security/tpotce/discussions/1056

**Reply by t3chn0m4g3** [reply]
There is no management traffic going *through* the honeypots, since management traffic is completely isolated, unreachable for the honeypots. For a honeypot to reach the management endpoint it would need a docker configuration  running the honeypot on the host network (bridged) instead on the docker network.
